TileStache/py3_compat.py

```diff
@@ -0,0 +1,28 @@
+# python3 compatibity while retaining checking
+# for both str and unicode in python2
+try:
+    unicode = unicode
```

Comment:
Instead of doing this, can you make a list?
`string_types = (str, unicode)` for python2 and `string_types = (str,)` for python3. I don't think we need `unicode` directly, just things like `is_instance(val, string_types)`